src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  if (days < 7) {
```
ты уже проверил что days >= 3